src/component/Filter/Filter.jsx

```diff
@@ -39,23 +38,24 @@ const Filter = () => {
           <h3 className="large-heading">Price</h3>
 
           <h6 className="text-md space-between">
-            <span id="min-price">₹ 200</span> <span id="max-price">₹1000</span>
+            <span id="min-price">₹ {priceRange}</span> <span id="max-price">₹ 66000</span>
           </h6>
           <input
             className="input range-input"
             id="price-range"
             type="range"
-            min="200"
-            max="1000"
-            step="50"
+            min="2000"
+            max="66000"
```

Comment:
These price attributes are hard coded. We should pass the state variable max={max_price}.

Reply:
Done